intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -10,22 +10,303 @@ \chapter{Preparing the Environment}
+      that you generated and saved above.
+
+  \includegraphics*[width=3.25in, height=3.13in, keepaspectratio=false]{ch2-putty-key-setup}
+
+    \item  In the category \menu[,]{Connection,SSH,X11}, click
+      the \keys{Enable X11 Forwarding} checkbox.
+
+  \includegraphics*[width=3.54in, height=3.41in, keepaspectratio=false]{ch2-putty-x-forwarding}
+
+    \item  Now go back to $<$Session$>$, and fill in ``\emph{\hpcname}'' in the
+      \keys{Saved Sessions} field and press \keys{Save} to
+      store the session information.
+
+  \includegraphics*[width=3.30in, height=3.17in, keepaspectratio=false]{ch2-putty-saved-session}
+
+    \item  Now pressing \keys{Open}, will open a terminal window and
+      asks for you passphrase.
+
+  \includegraphics*[width=3.94in, height=2.47in, keepaspectratio=false]{ch2-putty-enter-password}
+
+    \item  The first time you make a connection to the loginnode, a Security
+      Alert will appear and you will be asked to verify the authenticity of the
+      loginnode.
+      \ifgent
+      Make sure the fingerprint in the Alert matches \strong{\loginfingerprint}
+      \fi
+
+  \includegraphics*[width=3.91in, height=2.42in, keepaspectratio=false]{ch2-putty-verify-authenticity}
+
+    \item  After entering your correct passphrase, you will be connected to the
+      login-node of the \hpc.
+    \item  To check you can now ``Print the Working Directory'' (pwd) and check
+      the name of the computer, where you have logged in (hostname):
+
+  \begin{prompt}
+  %\shellcmd{pwd}%
+  %\homedir%
+  %\shellcmd{hostname --f}%
+  %\loginhost%
+  \end{prompt}
+
+    \item  For future PuTTY sessions, just select your saved session (i.e.
+      ``\emph{\hpcname}'') from the list, \keys{Load} it and press
+      \keys{Open}.
+  \end{enumerate}
+
+%ENDIF windos
+\fi
+
+\ifmacORlinux
+\subsection{Connect}
+\label{sec:connect}
+
+The \hpc is only accessible from within the \university campus network, but you can
+get external access (e.g., from home) by using a VPN connection.
+
+Open up a terminal and enter the following command to connect to the \hpc.
+\ifmac
+You can open a terminal by navigationg to Applications and then Utilities in the finder, or enter Terminal in spotlight and select Terminal.app
+\fi
+
+\begin{prompt}
+%\shellcmd{ssh $<$%\userid%$>$@\loginnode}%
+\end{prompt}
+
+The first time you make a connection to the login node, you will be asked to
+verify the authenticity of the login node, e.g.,
+
+\begin{prompt}
+%\shellcmd{ssh \userid{}@\loginnode{}}%
+The authenticity of host %\loginnode% (<IP-adress>) can't be established.
+RSA key fingerprint is %\loginfingerprint%
+Are you sure you want to continue connecting (yes/no)? %\strong{\emph{yes}}%
+\end{prompt}
+
+Here, user \userid wants to make a connection to the ``\hpcname'' cluster at
+\university via the login node ``\loginnode''.
+
+\fi
+\strong{Congratulations, you're on the \hpc Super-Computer now!}
+Okay, I am on the supercomputer, but where am I?
```

Comment:
I'm not writing this text, merely reorganising. 
Another pass over all the text will be needed, I first want things to be consistent and correct, the rest can come later :p